NEWS.md

```diff
@@ -40,6 +40,7 @@ Standard library changes
 ------------------------
 
 * `startswith` now supports seekable `IO` streams ([#43055])
+* printing integral `Rational`s will try to avoid printing denominators ([#45396])
```

Comment:
This feels a bit vague verging on misleading. My understanding is that it will skip printing a denominator of 1 in a typed IO context.

Reply:
Would
```suggestion
* printing integral `Rational`s will skip the denominator in `Rational`-typed IO context (e.g. in `Arrays`) ([#45396])
```
be better?

Reply:
Yes!